docs/developers/build-from-source.md

```diff
@@ -20,7 +20,20 @@ Recommended:
 
 ## Dependencies
 
-- Make sure your system has __Go 1.16+__ installed. https://golang.org/doc/install
+Make sure your system has __Go 1.19+__ installed. https://golang.org/doc/install
+
+Step-by-step Instructions to install Go on Ubuntu. https://www.linuxtechi.com/install-go-golang-on-ubuntu-linux/
+
+```shell
+$ sudo apt update
+$ wget https://go.dev/dl/go1.21.4.linux-amd64.tar.gz # Update to the most current release version
+$ sudo tar -C /usr/local -xzf go1.21.4.linux-amd64.tar.gz # Update to the most current release version
+$ ls /usr/local/go
+$ nano ~/.bash_profile # PASTE the following, then SAVE: export PATH=$PATH:/usr/local/go/bin
+$ source ~/.bash_profile
+$ go version
+```
+
```

Comment:
All three of these paragraphs are Go installation instructions. I'd prefer to defer simply to one -- the official Go docs -- rather than provide 3 redundant options which will need updating. The important part here -- and thanks for updating it -- is the minimum Go version.